chaoscenter/web/src/views/ChaosProbes/ChaosProbeTable.tsx

```diff
@@ -116,27 +124,40 @@ const ChaosProbeTable = ({ content, refetchProbes }: ChaosProbesTableProps & Ref
       {
         Header: '',
         id: 'threeDotMenu',
-        Cell: ({ row }: { row: Row<Probe> }) => <MenuCell row={row} refetchProbes={refetchProbes} />,
+        Cell: ({ row }: { row: Row<Probe> }) => (
+          <MenuCell row={row} refetchProbes={refetchProbes} setEditProbe={setEditProbe} />
+        ),
         disableSortBy: true
       }
     ];
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
   return (
-    <TableV2<Partial<Probe>>
-      className={css.table}
-      columns={columns}
-      data={content}
-      sortable
-      onRowClick={rowDetails =>
-        rowDetails.name &&
-        history.push({
-          pathname: paths.toChaosProbe({ probeName: rowDetails.name }),
-          search: `probeName=${rowDetails.name}&tab=${ProbeTabs.EXECUTION_RESULTS}&infrastructureType=${rowDetails.infrastructureType}`
-        })
-      }
-    />
+    <>
+      <TableV2<Partial<Probe>>
+        className={css.table}
+        columns={columns}
+        data={content}
+        sortable
+        onRowClick={rowDetails =>
+          rowDetails.name &&
+          history.push({
+            pathname: paths.toChaosProbe({ probeName: rowDetails.name }),
+            search: `probeName=${rowDetails.name}&tab=${ProbeTabs.EXECUTION_RESULTS}&infrastructureType=${rowDetails.infrastructureType}`
+          })
```

Comment:
```suggestion
            search: new URLSearchParams({
                        probeName: rowDetails.name,
                        tab: ProbeTabs.EXECUTION_RESULTS,
                        infrastructureType: rowDetails.infrastructureType
                      })
```

this will urlify the params and escape issue causing characters

Reply:
There is a type mismatch `Type 'URLSearchParams' is not assignable to type 'string | { (regexp: string | RegExp): number; (searcher: { [Symbol.search](string: string): number; }): number; } | undefined'` 

Even if you scope it above and set it using a const, the type of URLSearchParams isn't compatible with search param